src/main/java/tsunami/security/scanner/utilities/App.java

```diff
@@ -16,12 +16,180 @@
+import java.util.List;
+import java.util.Map;
+
 public class App {
-    public String getGreeting() {
-        return "Hello world.";
-    }
+    public static void main(String[] args) throws IOException, ApiException, ClassNotFoundException {
+        /* Parse args read from command line */
+
+        ApplicationArgs jArgs = new ApplicationArgs();
+        JCommander helloCmd = JCommander.newBuilder()
+                .addObject(jArgs)
+                .build();
+        helloCmd.parse(args);
+
+        String appName = jArgs.getName();
+        String appVersion = jArgs.getVersion();
+
+        System.out.println("Hello " + appName + " " + appVersion);
+
+        /* Logic to get the relative application config */
+        String configPath = System.getProperty("user.dir") + "/application/" + appName + "/";
+        File file = new File(configPath);
+        if (file.isDirectory()) {
+            try {
+                File[] files = file.listFiles();
+                for (File f : files) {
+                    if (!f.isDirectory())
+                        /* TODO: parse all config files under a certain app name to Kubernetes Object here, maybe use index to represent each config file*/
+                        System.out.println(f);
+                }
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+        }
+
+
+        /* Creation of Pod and Service without config files
+        V1Pod pod =
+                new V1PodBuilder()
+                        .withNewMetadata()
+                        .withName("apod")
+                        .endMetadata()
+                        .withNewSpec()
+                        .addNewContainer()
+                        .withName("www")
+                        .withImage("nginx")
+                        .withNewResources()
+                        .withLimits(new HashMap<>())
+                        .endResources()
+                        .endContainer()
+                        .endSpec()
+                        .build();
+        System.out.println(Yaml.dump(pod));
+
+        Map<String,String> m = new HashMap<>();
+        Map<String,String> m2 = new HashMap<>();
+        V1Service svc =
+                new V1ServiceBuilder()
+                        .withNewMetadata()
+                        .withName("wordpress")
+                        .withLabels(m)
+                        .addToLabels("app", "wordpress")
+                        .endMetadata()
+                        .withNewSpec()
+                        .addNewPort()
+                        .withPort(80)
+                        .endPort()
+                        .withSelector(m2)
+                        .addToSelector("app","wordpress")
+                        .addToSelector("tier", "frontend")
+                        .withType("LoadBalancer")
+                        .endSpec()
+                        .build();
+        System.out.println(Yaml.dump(svc));
+        */
```

Comment:
Is this for future reference? If so, you can replace this with a comment that links to the api example. This helps to make the source code clean.